docs/contribute.rst

```diff
@@ -127,28 +127,28 @@ Server architecture
 - *URLs:* http://www-demo1.allizom.org/ , http://www-demo2.allizom.org/ and
   http://www-demo3.allizom.org/
 - *PHP SVN branch:* trunk, updated every 10 minutes
-- *Bedrock locale SVN branch:* trunk, updated every 10 minutes
+- *Bedrock locales dev repo:* master, updated via a webhook on pushes
```

Comment:
Technically only the new infrastructure is using the webhook, and the current infrastructure is still using a cron job. Even with the webhook deploys are not instantaneous, so as long as that expectation is not being set I think this is fine.